docs/how-to-work-on-coding-challenges.md

```diff
@@ -124,8 +124,8 @@ Every challenge needs an `id`. If you don't specify one, then MongoDB will creat
 
 To generate a new one in a shell (assuming MongoDB is running separately):
 
-1. Run `mongo` command
-2. Run `ObjectId()` command
+1. Run `mongo` command.
```

Comment:
I don't think a full stop is necessary for a numbered list. If the convention is this across other documents as well then this is fine.

Reply:
Yes, its a convention across all documents. Thanks, @hassaanp

Reply:
I would agree with Hassaan; unless the point/list-item is a sentence, I would expect there not to be a full-stop. In fact, there are full-stops I would not mind seeing being removed. `E.g.`
> - Simplify the challenge, or
> - Split the challenge into two challenges.